report-handling-procedure.md

```diff
@@ -0,0 +1,197 @@
+:point_left: [Come back to index](README.md)
+
+# Chicago Python Users Group Code of Conduct
+
+The following guide was borrowed almost-entirely from the excellent Django Code
+of Conduct [Report Handling
+Procedure](https://github.com/django/code-of-conduct/blob/master/reports.md).
+
+## Report handling procedure
+
+This document aims to present the way we handle received reports. This is not a
+complete guide, but a set of guidelines or best practices that guide the way we
+provide support to the community. Each issue is different and requires
+individual consideration on a case-to-case basis.
+
+### How we receive reports
+
+All reports received by us are usually sent to the
+[chipy.conduct@gmail.com](mailto:chipy.conduct@gmail.com) email address
+that is automatically forwarded to the on-call receivers of complaints.
+
+It is possible for an individual to receive a report individually via email,
+Slack or in-person. They should be responsible for summarizing and emailing the
+central email account where possible (i.e. there is no conflict of interest).
+
+### On-call duty
+
+In order to achieve a fast response time to received reports, we implement a
+month long on-call duty that is rotated between Organizers. On the first
+meeting of each month, we assign one member to be the primary person on duty
+and another member to be a secondary supporter. The schedule is managed via
+[On Call Roster](http://www.chipy.org/pages/conduct-on-call/).
```

Comment:
I'm think there is an open issues to add the CoC to the main menu. In addition to that, to make this more discoverable, should we link this from http://www.chipy.org/contact/ - which is currently on the menu.
Additionally to make in-person reporting possible, we should probably announce this at our events.